client/client-multi/client-cpp/Modules/FindBotan.cmake

```diff
@@ -25,28 +25,31 @@
 
 message("\nLooking for Botan C++ headers and libraries")
 
+set(BOTAN_VERSION_NAME botan-1.11) 
+set(BOTAN_LIB libbotan-1.11) 
+
 if(NOT WIN32)
     include(FindPkgConfig)
     if(PKG_CONFIG_FOUND)
-        pkg_check_modules(BOTAN botan-1.11)
+        pkg_check_modules(BOTAN ${BOTAN_VERSION_NAME})
     endif(PKG_CONFIG_FOUND)
 endif(NOT WIN32)
 
 # find the include files
 find_path(BOTAN_INCLUDE_DIR
-    botan/version.h
+    ${BOTAN_VERSION_NAME}/botan/version.h
     HINTS
     ${CMAKE_FIND_ROOT_PATH}/include
     ${BOTAN_INCLUDE_DIRS})
 
 # locate the library
 if(WIN32)
-    set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} botan)
+    set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} ${BOTAN_VERSION_NAME})
 else(WIN32)
     if(Botan_USE_STATIC_LIBS)
-        set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} libbotan.a libbotan-1.11.a)
+        set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} ${BOTAN_LIB}.a)
     else(Botan_USE_STATIC_LIBS)
-        set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} botan botan-1.11)
+        set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} ${BOTAN_VERSION_NAME})
```

Comment:
Don't like this line. `BOTAN_VERSION_NAME` is for directory name, it's just a coincidence the library is named the same way.